content/ops/updating-cf.md

```diff
@@ -0,0 +1,18 @@
+---
+menu:
+  main:
+    parent: ops
+title: Updating Cloud Foundry
+weight: 10
+---
+
+Note: Grab something to eat and drink, this might take a while and an update shouldn't be left unattended.
+
+1. SSH to jumpbox
```

Comment:
Actually, we need to add like an "AWS onbaording for Cloud Foundry operators" doc. Because the only way to SSH to the jumpbox is if you have the key to it...